### PR TITLE
Adding SU2-7.0.6 software

### PR DESCRIPTION
**Pull request by alirezaghavaminia**
Hello, 
I have written an `Easyconfig` file for the `SU2` software. 
Please check it and PM me if it requires any changes.
Thank you very much.

### CODE REVIEW

File: easybuild/easyconfigs/f/FPM/FPM-1.4.0-Ruby-2.7.1.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# By: Alireza Ghavaminia <a.ghavami@alumni.iut.ac.ir> at Isfahan University of Technology
```
Comment (Micket) [inline easybuild/easyconfigs/f/FPM/FPM-1.4.0-Ruby-2.7.1.eb:1]: The PR doesn't mention this file?

File: easybuild/easyconfigs/r/Ruby/Ruby-2.7.1.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# By: Alireza Ghavaminia <a.ghavami@alumni.iut.ac.ir> at Isfahan University of Technology
```
Comment (Micket) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.7.1.eb:1]: The PR doesn't mention this file either?

File: easybuild/easyconfigs/s/SU2/SU2-7.0.2-foss-2020a.eb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# easyconfig file for SU2-7.0.6
```
Comment (Micket) [inline easybuild/easyconfigs/s/SU2/SU2-7.0.2-foss-2020a.eb:1]: This is a incorrect duplicate file.

File: easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+description = """Computational analysis tools have revolutionized the way we design engineering systems, but
+most established codes are proprietary, unavailable, or prohibitively expensive for many users. The SU2 team
+is changing this, making multiphysics analysis and design optimization software freely available and involving
+everyone in its creation and development."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+github_account = 'su2code'
+source_urls = ['https://github.com/su2code/SU2/archive/']
```
Comment (Micket) [inline easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb:19]:
```suggestion
source_urls = [GITHUB_SOURCE]
```

File: easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['5be22a992952b08f16bb80658f6cbe29c62a27e20236eccd175ca58dbc4ed27d']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('ParaView', '5.8.0', '-Python-3.8.2-mpi'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2')
```
Comment (Micket) [inline easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb:26]:
```suggestion
    ('SciPy-bundle', '2020.03', verionsuffix)
```

File: easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb [hunk 6]
```diff
@@ -0,0 +1,57 @@
+# easyconfig file for SU2-7.0.6
+# By Alireza Ghavaminia a.ghavami@alumni.iut.ac.ir (Isfahan University of Technology)
+
+easyblock = 'ConfigureMake'
+
+name = 'SU2'
+version = '7.0.6'
```
Comment (Micket) [inline easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb:7]: This file should have a `versionsuffix = '-Python-%(pyver)s'`
(also should be included in the filename)

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb (hunk 5 above)
Comment (Micket) [reply easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb:26]: Ack, i made this in the wrong file. I mean the correct file (7.0.6)

File: easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb (hunk 6 above)
Comment (Micket) [reply easybuild/easyconfigs/s/SU2-7.0.2-foss-2020a.eb:7]: this as well -> 7.0.6

**Comment by boegel** [comment]
replaced with #11446

**Pull request closed without merging**